Source/ProjectFirma.Web/Views/Project/DetailViewData.cs

```diff
@@ -408,7 +408,7 @@ string treatmentAreaGridDataUrl
 
             ProjectAgreementByGrantAllocations = ProjectAgreementByGrantAllocation.MakeAgreementProjectsByGrantAllocation(Project.ProjectGrantAllocationRequests.ToList());
 
-            
+            IsLoaProgramProject = project.ProjectPrograms.Any(x => x.ProgramID == 3);

```

Comment:
What if we made this an extension method on the project? 
So then you could write myProject.IsProjectInLoaProgram and that would return a bool if it is or not.
I would also recommend making the "3" ID a constant on the Program class, in case we need to use it elsewhere

Reply:
Added the const to Program.cs. I put the bool property in Project.cs instead of ProjectModelExtensions.cs because it seems like that's where most of the other "Is" bools are located.